06-reactjs/components/input.js

```diff
@@ -0,0 +1,47 @@
+//LIBS
+var React = require('react');
+var _ = require('lodash');
+
+var Input = React.createClass({
+
+    propTypes: {
+        index: React.PropTypes.string.isRequired,
+        onChange: React.PropTypes.func.isRequired,
```

Comment:
onChange is really required? What about an input with uncontroller behavior, the onChange is nor really needed